src/Core/Extensions/DateTimeExtensions.cs

```diff
@@ -53,17 +65,20 @@ public static DateTime EndOfMonth(this DateTime self, CultureInfo culture)
     /// Returns the first day of the week.
     /// </summary>
     /// <param name="self"></param>
-    /// <param name="firstDayOfWeek"></param>
+    /// <param name="culture"></param>
     /// <returns></returns>
-    public static DateTime StartOfWeek(this DateTime self, DayOfWeek firstDayOfWeek)
+    public static DateTime StartOfWeek(this DateTime self, CultureInfo culture)
```

Comment:
You cannot remove the previous signature, maybe already used by many developers.

Reply:
May I add `[Obsolute]` attribute to it?

Reply:
Yes. It's for the other methods.

For public methods, you can't add `[Obsolete]` because in 95% of cases, it's a good value. You can create another signature with the `Culture`. But you have to manage both.